lib/debug/config.rb

```diff
@@ -479,12 +479,15 @@ def self.commands
   end
 
   def self.help
```

Comment:
This method is also used for generating `README.md` so its content shouldn't be colorized.

Reply:
Reverted changes.

Thank you.